samples/Caliburn.Metro.Demo/Caliburn.Metro.Demo.csproj

```diff
@@ -45,11 +25,35 @@
     <Reference Include="Caliburn.Micro.Platform">
       <HintPath>..\..\packages\Caliburn.Micro.2.0.0\lib\net40\Caliburn.Micro.Platform.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Threading.Tasks" Condition=" '$(Configuration)' == 'Net4-Debug' Or '$(Configuration)' == 'Net4-Release' ">
```

Comment:
@thoemmi no no no. Try adding a reference and see if that's the case.
This condition was generated, but that's not the point. This condition is only for the async package dlls, so that we can switch between configurations with no problems, this will never need to be changed.
Personally, I've been using this method in managing multi-targeting scenarios and this is my favorite by far. Please try the branch if you have any concerns.